### PR TITLE
Add API Support to attachment in Articles

### PR DESCRIPTION
**Pull request by kuisathaverat**
new API calls to upload and delete article's attachments, also it fix a bug when you are listing attachments from an article, the previous version, return articles instead of article's attachments.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
   <groupId>com.cloudbees.thirdparty</groupId>
   <artifactId>zendesk-java-client</artifactId>
-  <version>0.3.2-SNAPSHOT</version>
+  <version>0.3.3-SNAPSHOT</version>
```
Comment (christ66) [inline pom.xml]: Revert.

**Comment by christ66** [comment]
:-1: for the `pom.xml` change. Once that is fixed then it has my :+1:.

File: src/main/java/org/zendesk/client/v2/Zendesk.java [hunk 2]
```diff
@@ -486,6 +490,29 @@ public void deleteSuspendedTicket(long id) {
...
+    } else {
+      builder.addHeader("Authorization", "Bearer " + oauthToken);
+    }
+    builder.setHeader("Content-Type", "multipart/form-data");
+    builder.addBodyPart(
+        new FilePart("file", file, "application/octet-stream", Charset.forName("UTF-8"), file.getName()));
+    final Request req = builder.build();
+    return complete(submit(req, handle(ArticleAttachments.class, "article_attachment")));
+  }
+
```
Comment (ydubreuil) [inline src/main/java/org/zendesk/client/v2/Zendesk.java]: Could you use 4 spaces indentation, to match code style of the rest of this file?

**Comment by ydubreuil** [comment]
:+1: for me too. Could you squash your commit before I merge @kuisathaverat ? If you don't know how to do it, I'll show you, don't close the PR.

**Comment by christ66** [comment]
:+1: from me.

**Comment by ydubreuil** [comment]
Merged after squashing in https://github.com/cloudbees/zendesk-java-client/commit/6748bceda0f9c6617bf0df25cc1c4c8626a8e6dd

**Pull request closed without merging**